docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+* A ZIP archive of the XCFramework itself should be uploaded to a convenient file storage with direct access (for example,
+S3, GitHub Releases, or a Maven repository). Choose the option that is easiest to integrate into your workflow.
+* The `Package.swift` file describing the package needs to be placed in a Git repository. You need to decide which
+repository to use. Pros and cons for several options are discussed below.
+
+### Options for Package.swift distribution
+The `Package.swift` that describes the Swift Package needs to be placed in a Git repository:
+* Store the `Package.swift` file in an independent repository to make its versioning separate from the
+KMP project the file describes. This is the recommended approach, allowing scalability and generally easier
+maintenance.
+* Put the `Package.swift` file next to your KMP code for a more straightforward approach. Keep in mind that versioning
+a Swift Package in the same repository as the code can become challenging as SPM uses Git tags for versioning packages
+and this can conflict with using tags for the project itself. 
+* Store the `Package.swift` within the consumer project's repo to avoid the versioning and maintenance issues.
+
+  However, this approach can cause problems for multi-repository SPM setups of the consumer project and further automation:
+  1. Only one consumer package in a multi-package project can depend on the external module, to avoid dependency conflicts
+  within the project. So all the logic depending on your KMP module should be encapsulated in a particular consumer package.
+  2. If you publish the KMP project using an automated CI process, this process would need to include publishing the
+  updated `Package.swift` file to the consumer repo. Such a phase in CI can be difficult to maintain as it may lead
+  to conflicting updates of the consumer repo.
+
+Now that you understand the options, create or adjust a Git repository as needed.
+
+## Create the XCFramework and the Package.swift file
+
+> The following example assumes that the shared code of your KMP project is stored in the `shared` module.
+> If your project is structured differently, substitute "shared" in code and path examples with the name of your module.  
+>
+{type="tip"}
+
+To provide a Swift package, you can follow these steps:
+1. Set up publishing of an [XCFramework](multiplatform-build-native-binaries.md#build-xcframeworks). Add an `XCFramework`
+call to your iOS targets description in the `shared/build.gradle.kts` file:
+   ```kotlin
+   import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+   
+   kotlin {
+      // other KMP targets
+      // ...
+      val xcf = XCFramework()
+      // Name of the module that will be imported in the consumer project
+      val xcframeworkName = "Shared"
+   
+      listOf(
+         iosX64(),
+         iosArm64(),
+         iosSimulatorArm64(),
+      ).forEach {
+           it.binaries.framework {
+              baseName = xcframeworkName
```

Comment:
Could we please add the recommendation to specify the CFBundleIdentifier; this will avoid the K/N linker warning:
```
// Specify CFBundleIdentifier
binaryOption("bundleId", "org.example.${xcframeworkName}")
```

Reply:
Same [here](https://github.com/JetBrains/kotlin-web-site/pull/4012/files#diff-a04ec0321e644ade5a5ed2af850e02ff2dd29178e3d1662417a0bb057501abd7R145)